tests/test_pythia8_fill.py

```diff
@@ -0,0 +1,140 @@
+import chromo
+import numpy as np
+from chromo.common import EventData
+from pathlib import Path
+import importlib
+from .util import run_in_separate_process
+
+
+def init_pythia8():
+    lib = importlib.import_module(f"chromo.models._pythia8")
+    chromo_path = Path(chromo.__file__).parent
+    xml_path = chromo_path / "iamdata/Pythia8/xmldoc"
+    pythia = lib.Pythia(str(xml_path), False)
+    seed = 1
+    pythia.settings.resetAll()
+    pythia.readString("Random:setSeed = on")
+    pythia.readString(f"Random:seed = {seed}")
+    pythia.readString("Print:quiet = on")
+    pythia.readString("ProcessLevel:all = off")
+    pythia.readString("ParticleDecays:tau0Max = 1e100")
+    pythia.init()
+    return pythia
```

Comment:
Why are you doing this? You can test this functionality more easily on a Pythia8 instance. You can access the underlying Pythia object in the Pythia8 instance using private attributes. You can set extra options with the `config` argument to the Pythia8 init.

Reply:
By the way, this is how I initialize and use pythia8 shared library. I was trying to use Pythia8 instance, but it's initialization times are very long. It would be nice to have some kind of shortcut initialization (without kinematics and loading tables), when Pythia8 is used only as afterburner.

For this test I can change the code to use Pythia8 instance, but it will add another ~10 sec to tests.

Reply:
Where is the difference in loading time coming from? I think it is coming only from the standard settings which enable all SoftQCD processes. This requires computing the inelastic cross-section. You can use config argument to set your own settings and override the standard. Then the initialization should be as fast as the manual one you are doing here.
Correct me if I am wrong.

Reply:
I found the obstacle that didn't allow to initiate fast. I changed Pythia8 class constructor, so when no kinematics is passed it works in decay mode.